python-adaline.py

```diff
@@ -61,20 +61,22 @@ def step(x):
         ADALINE_OUTPUT = (input_item[0]*WEIGHTS[0]) + (input_item[1]*WEIGHTS[1]) + (input_item[2]*WEIGHTS[2])
 
         # Run ADALINE_OUTPUT through the step function
-        ADALINE_OUTPUT = step(ADALINE_OUTPUT)
+        #DALINE_OUTPUT = step(ADALINE_OUTPUT)
```

Comment:
NOTE: this code was written in order to provide students with vanilla ADALINE implementation.

Weighted input to an ADALINE node is typically squashed into an output of -1 or +1. This is why the step function is required here.